target/linux/mvebu/files/arch/arm/boot/dts/armada-385-linksys-rango.dts

```diff
@@ -340,18 +330,29 @@
 
 	gpio-leds {
 		compatible = "gpio-leds";
-		pinctrl-0 = <&power_led_pin &sata_led_pin>;
+		pinctrl-0 = <&power_led_pin &sata_led_pin &wlan_2g_led_pin &wlan_5g_led_pin>;
```

Comment:
wlan_2g_led_pin wlan_5g_led_pin you define in the follow up patch. So at this point this should end up with a build failure.